text_extensions_for_pandas/array/test_tensor.py

```diff
@@ -499,6 +499,28 @@ def test_inferred_type(self):
         arr = TensorArray([True, False, True])
         self.assertEqual(arr.inferred_type, "boolean")
 
+    def test_numpy_ufunc(self):
+        data = np.arange(10).reshape(5, 2)
+        arr = TensorArray(data)
+
+        # ufunc with number
```

Comment:
It would be a good idea to repeat these tests with the arguments in the opposite order, i.e. `np.add(1, arr)`.

Reply:
I didn't think it was necessary since the inputs are handled as a list, but I can add them - it wouldn't hurt